.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,15 @@
+blank_issues_enabled: true
+contact_links:
+    about: Ask questions and discuss with other scipy community members
+  - name: Stack Overflow
+    url: https://stackoverflow.com/questions/tagged/scipy
+    about: Please ask and answer usage questions on Stack Overflow
+  - name: User Mailing list
+    url: https://mail.python.org/mailman3/lists/scipy-user.python.org/
+    about: User-related questions and announcements on the mailing list
```

Comment:
Can we take this out? I'd prefer not to encourage more questions there - Stack Overflow is better.

Reply:
Ok sure, actually we don't have much ref to this list now. Is this ok? Do we want to slowly deprecate this list? My opinion is yes: less work and not much is happening there.